pgx-examples/versioned_so/.cargo/config

```diff
@@ -0,0 +1,3 @@
+[build]
+# Postgres symbols won't be available until runtime
+rustflags = ["-C", "link-args=-Wl,-undefined,dynamic_lookup"]
```

Comment:
We didn't catch this during review, but this should have been removed as it causes some conflicts with the workspace override. @eeeebbbbrrrr is fixing this up before release.

Reply:
yah.  totally our bad.

In a workspace crate environment like pgx is, you **only** need the `.cargo/config` at the root workspace crate.  If it's in any "child" crates, the compiler goes stupid and you spend all day trying to figure it out and then you're thankful @Hoverbear is on your team!